doc/nrf/releases_and_maturity/releases/release-notes-3.0.0-preview1.rst

```diff
@@ -625,7 +625,9 @@ Common Application Framework
 Debug libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`cpu_load` library:
```

Comment:
BTW, this entry should most likely go to https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Reply:
done